packages/jellyfish-transaction-builder/__tests__/txn/loan_container.ts

```diff
@@ -0,0 +1,16 @@
+import { MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontainers'
+
+export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
+  constructor () {
+    super(undefined, 'defi/defichain:HEAD-6fa1c06')
+  }
+
+  protected getCmd (opts: StartOptions): string[] {
+    const cmd = super.getCmd(opts).filter(cmd => cmd !== '-eunospayaheight=7') // temporary remove -eunospayaheight=7 due to invalid flag
```

Comment:
why need to remove `eunospayaheight` ?
what if we move this code to the beginning of the relevant test file?

Reply:
Removing eunospayaheight. Tested and it works the same. Will pushed.
This code is going to be removed after epic/loan merged to the master branch in C++ side, I guess we just temporarily put it here.

Reply:
Also, if move the code to the relevant test file. Then, we will duplicate the same code for all the loan test files.